platforms/ios/HelloCordova.xcarchive/Products/Applications/HelloCordova.app/Frameworks/Sonect.framework/Headers/SNCConfiguration.h

```diff
@@ -0,0 +1,65 @@
+//
+//  SNCConfiguration.h
+//  SonectSDK
+//
+//  Created by Bruno Sousa on 21/12/2018.
+//  Copyright © 2018 Sonect. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SNCConfiguration : NSObject
+
+/**
+ A 2 digit ISO country code i.e. "ch" for Switzerland
+ */
+@property (nonatomic, nullable, copy, readonly) NSString* alpha2CountryCode;
+
+/**
+ The user's preferred currency i.e. "CHF" for Swiss Francs
+ */
+@property (nonatomic, nullable, copy, readonly) NSString* currency;
+
+/**
+ Set allowed country codes.
+ Example: would be US and Switzerland
+ @[@1, @41]
+ */
+@property (nonatomic, nullable, copy, readonly) NSArray <NSNumber *> *allowedCountryCodes;
+
+/**
+ A default withdraw amount index in the Withdraw View Controller amount picker control.
+ This control also remembers the user's last picked value.
+ */
+@property (nonatomic, readonly) NSUInteger defaultWithdrawAmountIndex;
```

Comment:
Looks like we support even 15 but we don't atm

Reply:
This will throw an exception if you pass 15 ;)